ProjFS.Mac/PrjFSKext/PrjFSKext/KauthHandler.cpp

```diff
@@ -277,6 +277,30 @@ void KauthHandler_AbortOutstandingEventsForProvider(VirtualizationRootHandle pro
     Mutex_Release(s_outstandingMessagesMutex);
 }
 
+static void UseMainForkIfNamedStream(
+    // In+out params:
+    vnode_t& vnode,
+    // Out params:
+    bool&    putVnodeWhenDone)
```

Comment:
Minor, if we're treating `putVnodeWhenDone` as an out parameter we should always assign something to it before leaving the function.

Reply:
Ah yes, that should also be under in+out in the current iteration of the code.

Reply:
Now that the code for releasing the incoming vnode is removed, this can be a pure output, so I've set it to false in the else clause.